### PR TITLE
[Release] Groom changelog /1.16.0 beta.1

### PR DESCRIPTION
**Pull request by palatter**
# What

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/communication-react/CHANGELOG.beta.md [hunk 1]
```diff
@@ -4,73 +4,51 @@ This log was last generated on Tue, 23 Apr 2024 17:54:31 GMT and should not be m
...
-- Add endCallButton option ([commit](https://github.com/azure/communication-ui-library/commit/not available) by jiangnanhello@live.com)
-- Captions Kind Changed event and store captions Kind inside context ([commit](https://github.com/azure/communication-ui-library/commit/not available) by 96077406+carocao-msft@users.noreply.github.com)
-- Improved closed captions storybook based on feedback ([commit](https://github.com/azure/communication-ui-library/commit/not available) by 96077406+carocao-msft@users.noreply.github.com)
-- Dismiss drawer on reaction click on mobile ([commit](https://github.com/azure/communication-ui-library/commit/not available) by 2684369+JamesBurnside@users.noreply.github.com)
+
+### Rich Text Editor - Public Preview
+
+The Web UI RichTextSendBox and MessageThread components now support rich text formatting to generate HTML content for chat messages. This feature includes support for: Different text styles (bold, italic and underline)Bulleted and numbered listsChanging indentation for textAdding and updating tables
```
Comment (vhuseinova-msft) [inline packages/communication-react/CHANGELOG.beta.md:16]: Could you update the format to include a list of the supported functionality, like:
```suggestion
The Web UI RichTextSendBox and MessageThread components now support rich text formatting to generate HTML content for chat messages. This feature includes support for: 
- Different text styles (bold, italic and underline)
- Bulleted and numbered lists
- Changing indentation for text
- Adding and updating tables
```

File: packages/communication-react/CHANGELOG.beta.md [hunk 2]
```diff
@@ -4,73 +4,51 @@ This log was last generated on Tue, 23 Apr 2024 17:54:31 GMT and should not be m
...
+
+### Closed Captions - Public Preview
+
+We are excited to announce that Azure Communications Services has recently added the Closed Captions feature. With closed captions, developers can provide a textual representation of the audio content in their videos, making it easier for users who are deaf or hard of hearing to follow along. Developers can enable this functionality today through our composites (e.g CallComposite, CallWithChatComposite). Azure Communications Services Closed Captions supports: Enable and disable closed captions in the call, Change spoken language, Being able to use captions outside of teams scenario
+
 ### Improvements
+
 - Renamed "file" to "attachment" for File Sharing Upload Flow ([PR #4369](https://github.com/azure/communication-ui-library/pull/4369) by 109105353+jpeng-ms@users.noreply.github.com)
```
Comment (vhuseinova-msft) [inline packages/communication-react/CHANGELOG.beta.md:24]: Every changelog item should be started from noun or V1 (verb in the base form) to be consistent with the previous releases. It looks like mostly V2 are used in this release changelog

File: packages/communication-react/CHANGELOG.beta.md [hunk 3]
```diff
@@ -4,73 +4,51 @@ This log was last generated on Tue, 23 Apr 2024 17:54:31 GMT and should not be m
...
-- fix local tile switcher action ([PR #4447](https://github.com/azure/communication-ui-library/pull/4447) by 94866715+dmceachernmsft@users.noreply.github.com)
-- Fix position of raised hand icon in video tiles ([PR #4453](https://github.com/azure/communication-ui-library/pull/4453) by 79475487+mgamis-msft@users.noreply.github.com)
-- Fix exception thrown when joining an Teams Interop meeting that has an unsupported chat thread ([PR #4482](https://github.com/azure/communication-ui-library/pull/4482) by 2684369+JamesBurnside@users.noreply.github.com)
-- remove the pptlive ([PR #4374](https://github.com/azure/communication-ui-library/pull/4374) by 93549644+ShaunaSong@users.noreply.github.com)
-- Remove CC for Capabilities ([PR #4408](https://github.com/azure/communication-ui-library/pull/4408) by 97124699+prabhjot-msft@users.noreply.github.com)
-- Fix default constant maxWidth value for fluent chat components ([PR #4510](https://github.com/azure/communication-ui-library/pull/4510) by 98852890+vhuseinova-msft@users.noreply.github.com)
```
Comment (vhuseinova-msft) [inline packages/communication-react/CHANGELOG.beta.md:58]: THis one probably needs to be included in the release list

File: packages/communication-react/CHANGELOG.beta.md [hunk 4]
```diff
@@ -4,73 +4,51 @@ This log was last generated on Tue, 23 Apr 2024 17:54:31 GMT and should not be m
...
-- Fix an issue where all messages were re-rendered when a new message was sent ([PR #4451](https://github.com/azure/communication-ui-library/pull/4451) by 98852890+vhuseinova-msft@users.noreply.github.com)
-- Improve detecting if camera permission is available by using Permissions API in the StatefulCallClient ([PR #4472](https://github.com/azure/communication-ui-library/pull/4472) by 2684369+JamesBurnside@users.noreply.github.com)
-- Fixed Upload Issue in Chat Sample App ([PR #4487](https://github.com/azure/communication-ui-library/pull/4487) by 109105353+jpeng-ms@users.noreply.github.com)
-- ACS captions should not shown for stable/beta release ([PR #4356](https://github.com/azure/communication-ui-library/pull/4356) by 96077406+carocao-msft@users.noreply.github.com)
-- Fix bug where captions stuck at bottom ([PR #4462](https://github.com/azure/communication-ui-library/pull/4462) by 96077406+carocao-msft@users.noreply.github.com)
-- Reaction animation inconsistency for presentaiton mode burse case fixed. ([PR #4370](https://github.com/azure/communication-ui-library/pull/4370) by mbellah@microsoft.com)
```
Comment (vhuseinova-msft) [inline packages/communication-react/CHANGELOG.beta.md:49]: Should we include this too?

File: packages/communication-react/CHANGELOG.beta.md [hunk 5]
```diff
@@ -4,73 +4,51 @@ This log was last generated on Tue, 23 Apr 2024 17:54:31 GMT and should not be m
...
-- Aria label for sidepane header ([PR #4475](https://github.com/azure/communication-ui-library/pull/4475) by edwardlee@microsoft.com)
-- Fix issue where android user's videos would not be on when joining the call caused by a stream change ([PR #4461](https://github.com/azure/communication-ui-library/pull/4461) by 94866715+dmceachernmsft@users.noreply.github.com)
-- Changing the order of the reaction animation appearance. ([PR #4368](https://github.com/azure/communication-ui-library/pull/4368) by mbellah@microsoft.com)
```
Comment (vhuseinova-msft) [inline packages/communication-react/CHANGELOG.beta.md:52]: Should we include this too?

File: packages/communication-react/CHANGELOG.beta.md [hunk 6]
```diff
@@ -4,73 +4,51 @@ This log was last generated on Tue, 23 Apr 2024 17:54:31 GMT and should not be m
...
-- fix local tile switcher action ([PR #4447](https://github.com/azure/communication-ui-library/pull/4447) by 94866715+dmceachernmsft@users.noreply.github.com)
-- Fix position of raised hand icon in video tiles ([PR #4453](https://github.com/azure/communication-ui-library/pull/4453) by 79475487+mgamis-msft@users.noreply.github.com)
-- Fix exception thrown when joining an Teams Interop meeting that has an unsupported chat thread ([PR #4482](https://github.com/azure/communication-ui-library/pull/4482) by 2684369+JamesBurnside@users.noreply.github.com)
```
Comment (vhuseinova-msft) [inline packages/communication-react/CHANGELOG.beta.md:55]: Should we include this too?

File: packages/communication-react/CHANGELOG.beta.md [hunk 7]
```diff
@@ -4,73 +4,51 @@ This log was last generated on Tue, 23 Apr 2024 17:54:31 GMT and should not be m
...
-- Allowing capability changes only in connected screen ([PR #4409](https://github.com/azure/communication-ui-library/pull/4409) by mbellah@microsoft.com)
-- Reaction button background color fixes on click ([PR #4407](https://github.com/azure/communication-ui-library/pull/4407) by mbellah@microsoft.com)
-- add adapter options to JS bundles ([PR #4486](https://github.com/azure/communication-ui-library/pull/4486) by dmceachern@microsoft.com)
-- Remove PPTLive overlay ([PR #4446](https://github.com/azure/communication-ui-library/pull/4446) by 93549644+ShaunaSong@users.noreply.github.com)
-- Add Chat Attachments in Live Messages ([PR #4438](https://github.com/azure/communication-ui-library/pull/4438) by 109105353+jpeng-ms@users.noreply.github.com)
-- Update caption logic to only process current speaker and ignore interceptors until the current speaker stops talking ([PR #4488](https://github.com/azure/communication-ui-library/pull/4488) by 96077406+carocao-msft@users.noreply.github.com)
-- Solving reaction out of screen viewport issue ([PR #4412](https://github.com/azure/communication-ui-library/pull/4412) by mbellah@microsoft.com)
```
Comment (vhuseinova-msft) [inline packages/communication-react/CHANGELOG.beta.md:68]: Should we include this too?

File: packages/communication-react/CHANGELOG.beta.md [hunk 8]
```diff
@@ -4,73 +4,51 @@ This log was last generated on Tue, 23 Apr 2024 17:54:31 GMT and should not be m
...
+- Updated button label rem size to fit shorter text ([PR #4460](https://github.com/azure/communication-ui-library/pull/4460) by edwardlee@microsoft.com)
+- Allowed capability changes only in connected screen ([PR #4409](https://github.com/azure/communication-ui-library/pull/4409) by mbellah@microsoft.com)
+- Fixed Reaction button background color fixes on click ([PR #4407](https://github.com/azure/communication-ui-library/pull/4407) by mbellah@microsoft.com)
+- Added adapter options to JS bundles ([PR #4486](https://github.com/azure/communication-ui-library/pull/4486) by dmceachern@microsoft.com)
+- Removed PPTLive overlay ([PR #4446](https://github.com/azure/communication-ui-library/pull/4446) by 93549644+ShaunaSong@users.noreply.github.com)
+- Added Chat Attachments in Live Messages ([PR #4438](https://github.com/azure/communication-ui-library/pull/4438) by 109105353+jpeng-ms@users.noreply.github.com)
 - Added Missing File Sharing Exports ([PR #4419](https://github.com/azure/communication-ui-library/pull/4419) by 109105353+jpeng-ms@users.noreply.github.com)
```
Comment (vhuseinova-msft) [inline packages/communication-react/CHANGELOG.beta.md:48]: Do we need this? (If this is the first PR that includes refactoring)

File: packages/communication-react/CHANGELOG.beta.md [hunk 9]
```diff
@@ -4,73 +4,51 @@ This log was last generated on Tue, 23 Apr 2024 17:54:31 GMT and should not be m
...
-- Fix VideoGallery to not use optimal video count when there are no remote videos on ([PR #4364](https://github.com/azure/communication-ui-library/pull/4364) by 79475487+mgamis-msft@users.noreply.github.com)
-- Update communication-chat to 1.5.0-beta.2 ([PR #4445](https://github.com/azure/communication-ui-library/pull/4445) by 3941071+emlynmac@users.noreply.github.com)
```
Comment (vhuseinova-msft) [inline packages/communication-react/CHANGELOG.beta.md:71]: Should we include this too?

File: packages/communication-react/CHANGELOG.beta.md (hunk 2 above)
Comment (palatter) [reply packages/communication-react/CHANGELOG.beta.md:28]: Will update to the correct grammar

File: packages/communication-react/CHANGELOG.beta.md (hunk 4 above)
Comment (palatter) [reply packages/communication-react/CHANGELOG.beta.md:49]: Will double check if reactions are in beta

File: packages/communication-react/CHANGELOG.beta.md (hunk 5 above)
Comment (palatter) [reply packages/communication-react/CHANGELOG.beta.md:52]: Will double check if reactions are in beta

File: packages/communication-react/CHANGELOG.beta.md [hunk 10]
```diff
@@ -4,74 +4,63 @@ This log was last generated on Tue, 23 Apr 2024 17:54:31 GMT and should not be m
...
-- Allowing capability changes only in connected screen ([PR #4409](https://github.com/azure/communication-ui-library/pull/4409) by mbellah@microsoft.com)
-- Reaction button background color fixes on click ([PR #4407](https://github.com/azure/communication-ui-library/pull/4407) by mbellah@microsoft.com)
-- add adapter options to JS bundles ([PR #4486](https://github.com/azure/communication-ui-library/pull/4486) by dmceachern@microsoft.com)
-- Remove PPTLive overlay ([PR #4446](https://github.com/azure/communication-ui-library/pull/4446) by 93549644+ShaunaSong@users.noreply.github.com)
+- Allow capability changes only in connected screen ([PR #4409](https://github.com/azure/communication-ui-library/pull/4409) by mbellah@microsoft.com)
+- Fix Reaction button background color fixes on click ([PR #4407](https://github.com/azure/communication-ui-library/pull/4407) by mbellah@microsoft.com)
+- Add adapter options to JS bundles ([PR #4486](https://github.com/azure/communication-ui-library/pull/4486) by dmceachern@microsoft.com)
+- Removed PPTLive overlay ([PR #4446](https://github.com/azure/communication-ui-library/pull/4446) by 93549644+ShaunaSong@users.noreply.github.com)
```
Comment (vhuseinova-msft) [inline packages/communication-react/CHANGELOG.beta.md:57]:
```suggestion
- Remove PPTLive overlay ([PR #4446](https://github.com/azure/communication-ui-library/pull/4446) by 93549644+ShaunaSong@users.noreply.github.com)
```

**Review by vhuseinova-msft — APPROVED** [review]

**Review by carocao-msft — APPROVED** [review]

**Pull request merged**